stable/dex/templates/certificate.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: certmanager.k8s.io/v1alpha1
+kind: Certificate
+metadata:
+  name: dex
+  namespace: kubeaddons
+  annotations:
+    "helm.sh/hook": pre-install
+spec:
+  secretName: dex
+  issuerRef:
+    name: kubernetes-ca
+    kind: ClusterIssuer
+  commonName: dex
+  organization:
+    - Mesosphere
+  dnsNames:
+    - dex-kubeaddons
```

Comment:
This should be configurable, let's add it to `values.yaml`

Reply:
https://jira.mesosphere.com/browse/DCOS-59044